features/step_definitions/moderation_steps.rb

```diff
@@ -49,7 +49,7 @@
 end
 
 Then /^the petition is not available for searching or viewing$/ do
-  step %{I search for "rejected" petitions with "#{@petition.action}"}
+  step %{I search for "Rejected" with "#{@petition.action}"}
   step %{I should not see the petition "#{@petition.action}"}
 end
```

Comment:
These two are inconsistent.  Should it be `I search for "Rejected" with "#{@petition.action}"` or `I search for "Rejected petitions" with "#{@petition.action}"}`, it's confusing if they're not the same.